packages/manager/modules/exchange/src/account/add/account-add.controller.js

```diff
@@ -223,12 +230,35 @@ export default class ExchangeAccountAddController {
       })
       .catch((error) => {
         this.messaging.writeError(
-          this.$translate.instant('exchange_ACTION_add_account_error_message'),
+          this.$translate.instant('exchange_ACTION_add_account_error_message', {
+            message: error.message || error.data?.message,
+          }),
           error,
         );
       })
       .finally(() => {
         this.hide();
       });
   }
+
+  onPasswordConfirmationChange() {
+    if (this.newAccountForm.passwordConfirmation.$error.required) {
+      this.newAccountForm.passwordConfirmation.$setValidity(
+        'isDifferentToPassword',
+        true,
+      );
+    } else {
+      this.newAccountForm.passwordConfirmation.$setValidity(
+        'isDifferentToPassword',
+        this.newAccount.password === this.newAccount.passwordConfirmation,
+      );
+    }
```

Comment:
**suggestion:** 
```
const valid = this.newAccountForm.passwordConfirmation.$error.required ? true : this.newAccount.password === this.newAccount.passwordConfirmation;
this.newAccountForm.passwordConfirmation.$setValidity(
  'isDifferentToPassword',
  valid,
);
```

Reply:
Thanks about this suggestion Anoop.
This code was already here before, probably to applied in another refactoring task.

Thanks.